explore-cardano/understanding-the-delegation-mechanism.md

```diff
@@ -1,5 +1,27 @@
 # Understanding the Delegation Mechanism
 
-As Cardano is a proof-of-stake system, owning ada not only allows you to buy goods or services, but also confers upon you the right and obligation to participate in the protocol and create blocks. 
+The stake delegation is a mechanism added to the Cardano proof of stake (PoS) protocol that allows the protocol to scale even in a setting where the set of stakeholders is highly fragmented.
```

Comment:
this sentence doesn't make sense. 

Revert to the existing sentence: As Cardano is a proof-of-stake system, owning ada not only allows you to buy goods or services, but also confers upon you the right and obligation to participate in the protocol and create blocks.